### ISSUE TITLE
iso_url for ubuntu 2004 is not working

### ISSUE THREAD

**Issue by jiatongw** [kubernetes-sigs/image-builder#247]
The ISO URL for ubuntu 2004 is not working anymore
```
==> vsphere: Retrieving ISO
==> vsphere: Trying http://cdimage.ubuntu.com/ubuntu-server/daily/20200107/focal-server-amd64.iso
==> vsphere: Trying http://cdimage.ubuntu.com/ubuntu-server/daily/20200107/focal-server-amd64.iso?checksum=sha256%3A4901eeba3ffaa56b9356bb496d161ade5bd1e2df579552aa6c40f33984a34614
==> vsphere: Download failed bad response code: 404
==> vsphere: error downloading ISO: [bad response code: 404]
Build 'vsphere' errored: error downloading ISO: [bad response code: 404]
```

Replace with this one? http://cdimage.ubuntu.com/releases/20.04/release/
I can make a PR for this.

**Comment by jiatongw** [comment]
ops, this release page http://cdimage.ubuntu.com/releases/20.04/release/ doesn't have ISO for `amd64` arch..

**Comment by jiatongw** [comment]
fixes by #254
